.devcontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Note: You can use any Debian/Ubuntu based image you want. 
+# Note: You can use any Debian/Ubuntu based image you want.
```

Comment:
I would be cautious with such statements. Have you actually tested with Trusty Thar?